.github/workflows/build.yml

```diff
@@ -4,6 +4,7 @@ on:
     branches:
       - main
       - beta
+      - windows-arm
```

Comment:
probably should be removed before this is merged

Reply:
Thanks, removed